README.md

```diff
@@ -58,10 +59,6 @@ cd mentat
 pip install -e .
 ```
 
-## Selecting which LLM Model to use
-
-We highly recommend using the default model, `gpt-4-1106-preview`, as it performs vastly better than any other model benchmarked so far. However, if you wish to use a different model, jump [here](#alternative-models).
-
 ### Add your OpenAI API Key
 
 You'll need to have API access to GPT-4 to run Mentat. There are a few options to provide Mentat with your OpenAI API key:
```

Comment:
Maybe this should be changed; everybody has gpt-4 access now, and it isn't even necessary anymore.